lms/static/sass/inline-discussion.scss

```diff
@@ -0,0 +1,22 @@
+// From lms-main-v1.scss
```

Comment:
Thanks. I'd remove the comment now, and replace it with a header that says what this file is.